pkgs/build-support/libredirect/default.nix

```
@@ -1,5 +1,4 @@
-{ stdenv, lib, coreutils }:
-
+{ darwin, llvmPackages_13, stdenv, lib, coreutils }:
```

We almost never import `darwin` as a whole. Instead import just the needed parts:
```suggestion
{ lib, stdenv, binutils-unwrapped, llvmPackages_13, coreutils }:
```
This will require placing `inherit (darwin) binutils-unwrapped;` to the all-packages.nix. This allows to ensure an adequate interface for doing overrides.

---

Unfortunately there is already a top-level `binutils-unwrapped`.

---

That is perfectly fine, in this case we probably want to pick Apple's implementation.